packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -450,6 +437,66 @@ export default class VTable {
         }
         return result;
     }
+
+    private setEmptyTableCells() {
+        for (let i = 0, row; (row = this.table.rows[i]); i++) {
+            for (let j = 0, cell; (cell = row.cells[j]); j++) {
+                if (cell) {

```

Comment:
cell will be undefined when j == 0. Is that expected?

Reply:
It will do cell = row-cells[j] before evaluating the cell when j == 0, so it won't be undefined.